32blit/engine/menu.hpp

```diff
@@ -127,20 +131,56 @@ namespace blit {
         screen.pen = selected_item_background;
         screen.rectangle(item_rect);
       }
-      
+
       screen.pen = foreground_colour;
       item_rect.x += item_padding_x;
       item_rect.y += item_adjust_y;
       item_rect.h += font.spacing_y; // adjust for alignment
       screen.text(item.label, font, item_rect, true, TextAlign::center_left);
     }
 
+    //
+    // Can be used in a derived class for global updates to the entire menu
+    //
+    virtual void update_menu(uint32_t time) {}
```

Comment:
Hmm, `Menu::update_menu` is a bit weird, but I don't have a better idea...

Reply:
I wanted it separate from update, but did not want to override update itself. This was the term I came up with. 🤷🏻‍♂️